patches/common/chromium/can_create_window.patch

```diff
@@ -172,49 +163,40 @@ index 09a26d8a7a6c..ffc823d49817 100644
 +      content::mojom::DataElementPtr ptr = content::mojom::DataElement::New();
 +      ptr->type = element.type();
 +      switch (element.type()) {
-+        case storage::DataElement::TYPE_BYTES: {
++        case network::DataElement::TYPE_BYTES: {
 +          ptr->bytes = std::string(element.bytes(), element.length());
 +          break;
 +        }
-+        case storage::DataElement::TYPE_BYTES_DESCRIPTION: {
-+          ptr->length = element.length();
-+          break;
-+        }
-+        case storage::DataElement::TYPE_FILE: {
++        case network::DataElement::TYPE_FILE: {
 +          ptr->path = element.path();
 +          ptr->offset = element.offset();
 +          ptr->length = element.length();
 +          ptr->expected_modification_time = element.expected_modification_time();
 +          break;
 +        }
-+        case storage::DataElement::TYPE_FILE_FILESYSTEM: {
-+          ptr->filesystem_url = element.filesystem_url();
++        case network::DataElement::TYPE_RAW_FILE: {
```

Comment:
`network::DataElement::TYPE_RAW_FILE` should be ignored, we don't have data writer setup for it.

```
case network::DataElement::TYPE_RAW_FILE:
case network::DataElement::TYPE_DATA_PIPE:
case network::DataElement::TYPE_UNKNOWN:
        NOTREACHED();
        break;
```